core/frontend/src/components/utils/BrIframe.vue

```diff
@@ -24,8 +29,21 @@
 <script lang="ts">
 import Vue from 'vue'
 
+import { basic2dVertexShaderSource, bubbles2dFragmentShaderSource } from '../../utils/shaders'
```

Comment:
```suggestion
import { basic2dVertexShaderSource, bubbles2dFragmentShaderSource } from '@/utils/shaders'
```

Reply:
Added both changes, the qualifiers names to complete attribute, location and uniforms and also the import path!

About "how it make faster" right now its not making that much difference, but this will serve as groundwork for future splash screen, and in that scenario it will probably shipped together with index making  the splash screen appears as soon as possible.